nipoppy/workflows/pipeline.py

```diff
@@ -373,7 +413,7 @@ def run_cleanup(self):
         """Log a summary message."""
         if self.n_total == 0:
             self.logger.warning(
-                "No participant-session pairs to run. Make sure there are no mistakes "
+                "No participants/sessions to run. Make sure there are no mistakes "
```

Comment:
Nitpick, but I would put "or" instead of "/"
https://www.plainlanguage.gov/guidelines/conversational/dont-use-slashes/